src/library/trackmodel.h

```diff
@@ -203,6 +203,11 @@ class TrackModel {
     virtual void select() {
     }
 
+    /// @brief modelKey returns a unique identifier for the model
+    /// @param noSearch don't include the current search in the key
+    /// @param baseOnly return only a identifier for the whole subsystem
+    virtual QString modelKey(bool noSearch = false) const = 0;
```

Comment:
Do we really need a default parameter? I don't recommend to use default parameters in virtual function declarations. Also applies to all implementations where this default is repeated over and over again.

Reply:
I guess we don't. Any possible alternative request besides that from WTrackTableView is hypothetical.